packages/reakit/src/Stack/__tests__/Stack-test.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { render } from "react-testing-library";
+import Stack from "../Stack";
+
+test("HTML attributes", () => {
+  const { getByTestId } = render(
+    <Stack id="test" aria-label="test" data-testid="test" />
+  );
+
+  expect(getByTestId("test")).toHaveAttribute("id", "test");
+  expect(getByTestId("test")).toHaveAttribute("aria-label", "test");
+});
+
+test("Default", () => {
+  const { container } = render(<Stack />);
+  expect(container.firstChild).toMatchSnapshot();
```

Comment:
I think we usually use `toMatchInlineSnapshot` as a convention in this library 💅🏻